hypha/static_src/src/sass/apply/components/_list.scss

```diff
@@ -44,16 +44,16 @@
         &--has-reviewed {
             display: grid;
             justify-content: space-between;
-            grid-template-columns: 40% 40% 20%;
+            grid-template-columns: 40% 30% 20%;
         }
 
         &--reviewer-name {
             max-width: 190px;
             overflow: hidden;
             font-weight: $weight--bold;
             text-overflow: ellipsis;
-            display: flex;
             align-items: center;
+            white-space: nowrap;
```

Comment:
After applying the `truncate` class the html element itself, we can get rid of the following rules from here -- `overflow`, `text-overflow` and `white-space: nowrap`.

Since you are removing the `display: flex`, you can also remove the `align-items: center` from here.